src/shared/data/UpcomingEvents.js

```diff
@@ -0,0 +1,10 @@
+// upcoming event's template: 
+// { image: EVENTS_22_23[0], title: "SEA Pit Night", date: "Wednesday, April 8 | 9PM-1AM", loc: "The Pit Pub @ UBC", link: "google.com"},
+
+import { EVENTS_22_23 } from "assets"
+
+const UPCOMING_EVENTS = [
+    { image: EVENTS_22_23[0], title: "SEA Pit Night", date: "Wednesday, April 8 | 9PM-1AM", loc: "The Pit Pub @ UBC", link: "google.com"},
```

Comment:
```suggestion
    {},
```
Don't forget to remove this! Thank you for adding this.